routes/Mojang.js

```diff
@@ -1,901 +0,0 @@
-const request = require('request');
-const net = require('net');
-const NodeCache = require('node-cache');
-
-const db = require('./../db-utils/DB_Mojang');
-
-const Utils = require('./../utils');
-
-const SKIN_STEVE = require('fs').readFileSync('./storage/static/steve.png'),
-  SKIN_ALEX = require('fs').readFileSync('./storage/static/alex.png');
-
-//FIXME: UUID '3126ba4c6uud424c877d1347fa974d23' belongs to user '_' but is not a valid UUID - API throws 500 because it detects it as invalid UUID (But should return the profile)
```

Comment:
Can't find that UUID - Looks like it changed because the User `_` now has `d324567b-6c91-4cee-8c30-872946b5a629`
But the API is fine with that and does not encounter any errors